llama-index-integrations/postprocessor/llama-index-postprocessor-voyageai-rerank/llama_index/postprocessor/voyageai_rerank/base.py

```diff
@@ -39,7 +39,7 @@ def __init__(
             )
 
         self._client = Client(api_key=api_key)
-        super().__init__(top_n=top_k, model=model, truncation=truncation)
+        super().__init__(top_k=top_k, model=model, truncation=truncation)
```

Comment:
actually, now that I'm seeing this, I'm pretty sure all our other rerankers use the `top_n` kwarg 👁️

Reply:
I changed to top_n